recon_linux.go

```diff
@@ -43,17 +44,16 @@ func (irc *IrcCon) StartUnixListener() {
 }
 
 // Attempt to hijack session previously running bot
-func (irc *IrcCon) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	unaddr, err := net.ResolveUnixAddr("unix", irc.unixastr)
 	if err != nil {
-		irc.Log(LWarning, "could not resolve unix socket")
+		irc.Error("could not resolve unix socket")
 		return false
 	}
 
 	con, err := net.DialUnix("unix", nil, unaddr)
 	if err != nil {
-		fmt.Println("Couldnt restablish connection, no prior bot.")
-		fmt.Println(err)
+		irc.Error("Couldnt restablish connection, no prior bot:", err)
```

Comment:
Basically my plan here is to make a separate process that proxies the connection, then you can kill the bot and/or restart it without dropping the connection at all. Not sure if that is for everyone, but you can't just send the FD for a TLS conn over like this.